lib/Bootstrap.php

```diff
@@ -226,7 +221,8 @@ public static function kernel(): Kernel|\App\Kernel|KernelInterface
         if ($debug) {
             umask(0000);
             Debug::enable();
-            @ini_set('display_errors', 'On');
+        } else {
+            ErrorHandler::register(new ErrorHandler(new BufferingLogger(), false));
```

Comment:
Are we sure this is necessary? The error handler should in theory be auto-registered by Symfony, no? The only exception is the Debug::enable() above, AFAIK.

Reply:
For me it looks like we need it, because without the registration of the ErrorHandler the error message looks like this:

![image](https://user-images.githubusercontent.com/104770750/220594334-0cf9f0fc-d8f2-45fe-8247-87efc6cb7650.png)

Reply:
For me it doesn't seem so, because when I register it. I got this error page:
![image](https://user-images.githubusercontent.com/104770750/220596011-613f7299-80d9-4bf0-8d54-b0a2a862ce8a.png)

Reply:
But in a regular Symfony app (even before Symfony Runtime), you didn't need to register it explicitly and would still get that same error in prod. My worry is, if we do enable it explicitly, this will interfere with Symfony doing it if and when Pimcore introduces #10344.

Reply:
Maybe this is related? symfony/symfony#49275

Reply:
I removed the registration in the else